src/main/java/com/google/sps/servlets/MemesServlet.java

```diff
@@ -87,5 +92,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String imgTag = String.format("<img src=\"%s\" width = \"300\"/>", blob.getMediaLink());
       response.getWriter().println(imgTag);
     }
+  }
+
+/** This funtions  randomly  get a string  captions from an array of string. When the function is called
+ * it randomly return memes caption from an array
+ * input: null
+ * return: strings..
+ * 
+ */
+  
+
+  public static String getCpation(){
+//     initilaise an array  of captions
+        final String[] proper_noun = {"Fred", 
```

Comment:
nit: variable names in Java are typically lowerCamelCase, not separated by underscores. Also consider naming this more specifically, like `captions` or `captionsList`, instead of proper_noun.